internal/transport/transport.go

```diff
@@ -19,6 +19,9 @@ type Client struct {
 	// Notification holds the settings value for if we have notifications enabled or not.
 	Notifications bool
 
+	// Update Radio set automatic updates or not
+	UpdateRadio bool
```

Comment:
UpdateRadio does not specify what the setting is. I think a better name would make it more clear.
```suggestion
	// CheckUpdatesOnStartup holds the setting values for if we should check updates on startup.
	CheckUpdatesOnStartup bool
```